src/main/ts/utils/Git.ts

```diff
@@ -117,3 +117,10 @@ const detectGitUrlFromDir = async (dir: string): Promise<string> => {
 
 export const resolveGitUrl = async (gitUrlArg: Option<string>, workingDirArg: string): Promise<string> =>
   gitUrlArg._tag === 'Some' ? gitUrlArg.value : await detectGitUrlFromDir(workingDirArg);
+
+export const hasLocalChanges = async (workingDir: string, branchName: string) => {
+  const git = gitP(workingDir);
```

Comment:
Do you want to also check if the workspace is dirty?

Reply:
nvm I see that's done elsewhere.

Reply:
Am checking that now